packages/ui-extensions-react/src/surfaces/point-of-sale/hooks/index.ts

```diff
@@ -14,6 +14,7 @@ export {
   useStatefulSubscribableLocale,
   destroyStatefulSubscribableLocale,
 } from './locale-api';
+export {useOrder} from './order-api';
```

Comment:
The AI created this export but there is no matching order-api file. Not that we want this hook anyway